Improvements/greet.js

```diff
@@ -0,0 +1,3 @@
+function greet(name) {
+  return name ? `hello ${name}` : `hello`;
```

Comment:
You can use normal single quote or double quote as they represent simple strings